packages/adminrouter/buildinfo.json

```diff
@@ -9,7 +9,7 @@
       "adminrouter": {
           "kind": "git",
           "git": "https://github.com/dcos/adminrouter.git",
-          "ref": "5e23b06be6c061b394d2123424675c542072a6f2",
+          "ref": "44bb0b2d119154f9b4ea9f1c3af184c278bb39b5",
```

Comment:
I am open to suggestions here, I am not sure how to fix it. We need the changes in that commit to expose Cosmos correctly. Do you want me to change the AdminRouter in a different PR?